### ISSUE TITLE
Modify the structure of the messages delivered to the display

### ISSUE THREAD

**Issue by jmaldonadof** [IntegratedAlarmSystem-Group/ias-webserver#22]
Currently the Webserver sends messages with the alarms to the display with the following structure:
```
{ 'pk': 'None', 'model': 'alarms.alarm', 'fields':{ 'core_id':...}}
```
In the case of lists, it sends a list of dictionaries. That structure was maintained since the beginning, when the webserver was using the model binding functionality. At this time, the dependency of the alarm to the django models will be disappear, so there is no reason to keep the messages structure in that way. 

The consumer must send, either an alarm converted to dict or list of dictionaries of alarms, removing the keys: pk, model and fields.

**Comment by acaproni** [comment]
Justine, did you investigating if it is the case to use a serialization tool like [apache avro](https://avro.apache.org/)? I am actually thinking to introduce avro in the core ([#78](https://github.com/IntegratedAlarmSystem-Group/ias/issues/78)). It also offers binding for different programming languages and among them java and python.

**Comment by jmaldonadof** [comment]
Alessandro, the problem mentioned in the issue is only part of the refactoring that we are doing to the webserver. It does not affect to the core, it only affects the communication between the webserver and the GUI. The idea is to simplify the message, because as a result of previous intents to use django tools which are not necessary anymore, the messages in this moment have unnecessary fields that we want to remove. But it is not something complicated, I only added the ticket to remember it and to have a reference to the description of the problem.

**Comment by acaproni** [comment]
Thanks for the explanation, jazmine. 
Even if not related to the core, apache avro is an option IMO.

**Comment by mxolivares** [comment]
Closed with (9907172e)
